### ISSUE TITLE
My Adopted Child isn’t mine anymore :(

### ISSUE THREAD

**Issue by justanordinaryarmy** [Luke100000/minecraft-comes-alive#660]
Hi love the mod! I just stumbled upon an issue on MCA reborn.

 In file version 7.4.7+1.18.2 for forge, there was this bug where I Adopted a child when they were a teen and later once they were an adult It shows that I’m not related to them anymore and doesn’t block romantic options because it doesn’t recognize me as the parent anymore. And for some weird reason when I click on my adopted child’s family tree it shows my promised partner as the parent, but with a  different person he was promised to, and not me. Even though the other random person was never promised with my partner. 
 
Tldr: my promise partner stole my adopted child with some other random dead person that he was never once promised with lol
